### ISSUE TITLE
Remove branding feature

### ISSUE THREAD

**Issue by revabec122** [calcom/docker#218]
https://github.com/calcom/cal.com/pull/7185

Does the above update mean that certain team features like removing Cal.com branding have been enabled for self-hosters with the 2.6.2 update?

If the answer's yes, that particular feature doesn't seem to be working for me with the docker install. Its a fresh install with the image being pulled from the most recent release and I have checked the box to remove branding. Despite that, I see "Cal.com" in the title and content of booking pages.

I have also tried adding the below variables to the .env file. Interestingly, I see the revised app name in the title of the booking page for a split second before it changes back to Cal.com.

NEXT_PUBLIC_APP_NAME=
NEXT_PUBLIC_SUPPORT_MAIL_ADDRESS=
NEXT_PUBLIC_COMPANY_NAME=

**Comment by krumware** [comment]
I don't think that has been fully completed yet. @PeerRich may be able to provide insight
